borg/testsuite/helpers.py

```diff
@@ -259,6 +259,40 @@ def testInvalidUnicode(self):
         assert e.match(str(b"ba\x80/foo", 'latin1'))
 
 
+@pytest.mark.parametrize("lines, expected", [
+    ([], None),
+    (["# Comment only"], None),
+    (["*"], []),
+    (["# Comment",
+      "*/something00.txt",
+      "  whitespace\t",
+      "/whitespace/at/end of filename \t ",
+      # Whitespace before comment
+      " #/ws*",
```

Comment:
ok, so this means we have significant whitespace at start and end of line.

i rather tend to not do this (although one could, but that would be a bit "what you get is what you deserve").
so how about stripping the lines and if somebody still wants a blank at beginning or end, it could be done later with a regex?